buildenv/jenkins/variables/defaults.yml

```diff
@@ -152,3 +152,26 @@ linux_x86-64_cmprssptrs:
       8: 'hw.arch.x86 && sw.os.ubuntu.16'
       9: 'hw.arch.x86 && sw.os.ubuntu.16'
       10: 'hw.arch.x86 && sw.os.ubuntu.16'
+#========================================#
+# Linux x86 64bits (large heap)
+#========================================#
+linux_x86-64_cmprssptrs:
+  boot_jdk:
+    8: '/usr/lib/jvm/java-7-openjdk-amd64'
+    9: '/usr/lib/jvm/adoptojdk-java-80'
+    10: '/usr/lib/jvm/adoptojdk-java-90'
+  release:
+    8: 'linux-x86_64-normal-server-release'
```

Comment:
How is this value used?  It's identical between this build and the compressed refs one.

Reply:
see https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/test
line 23-34